### PR TITLE
fix: footer legal links don't break 2 lines

### PR DESCRIPTION
**Pull request by victordelval**
## Description

Closes: regen-network/regen-registry#1434

* Also fix legal links in registry app. Now reads only from globals in web-components

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1636--regen-registry.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by victordelval** [comment]
@erikalogie ready for review, thanks!

**Comment by erikalogie** [comment]
@victordelval looks good!

**Review by mhagel — APPROVED** [review]
Thanks!

File: web-registry/src/globals.ts [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import {
-  URL_REGISTRY_MARKETPLACE_LEGAL,
-  URL_REGISTRY_TERMS_SERVICE,
-  URL_WEB_PRIVACY,
-} from 'web-components/lib/utils/globals';
-
-export const URL_PRIVACY = URL_WEB_PRIVACY;
-export const URL_TERMS_SERVICE = URL_REGISTRY_TERMS_SERVICE;
-export const URL_MARKETPLACE_LEGAL = URL_REGISTRY_MARKETPLACE_LEGAL;
```
Comment (flagrede) [inline web-registry/src/globals.ts:9]: Not directly related by this PR but these constants should have been created in registry in the first place.
It feels weird to import this kind of data from our component library.

**Review by flagrede — APPROVED** [review]
Fix looks good !
If you are willing to move related web-components globals to registry would be great otherwise we'll do it later.

File: web-registry/src/globals.ts (hunk 1 above)
Comment (victordelval) [reply web-registry/src/globals.ts:9]: Yeah, I had that in mind too. I thought that one of these global variables was being used on the website as well. I just checked that it's not, so I make this change of bringing the global variables to the registry

Comment (victordelval) [reply web-registry/src/globals.ts:9]: @flagrede done! Had to move a few components to the registry also

**Comment by victordelval** [comment]
@Mergifyio backport redwood hambach

**Comment by blushi** [comment]
@victordelval should this be backported into master too?

**Pull request merged**